processes/managing-reviewers.md

```diff
@@ -0,0 +1,49 @@
+# Managing Reviewers
+
+## Status
+
+implementable
+
+## Summary
+
+After the migration to use OWNERS files to manage reviewers, it is
+easier for us to add reviewers to separate repositories. This document
+describes the process for adding reviewers to a metal3 project repo.
+
+### Goals
+
+1. Describe a process for adding reviewers.
+2. Keep the process light-weight.
+
+### Non-Goals
+
+1. Change the process for adding [maintainers](../maintainers/).
+
+## Proposal
+
+Anyone can propose a patch to update an OWNERS file in a repository to
+add a reviewer. The patch should be submitted as a standalone PR,
+rather than being linked to any other contribution.
+
+The reviewer list for each repository will be pruned over time to
+remove contributors who are inactive.
+
+Reviewers may also be removed for behaving in a manner that other
+maintainers view as detrimental to the project, following the process
+described for maintainers in [Revoking Approval
+Access](../maintainers#revoking-approval-access).
+
+Two contributors with approval permission for the repository are
+required to approve any PR to add or remove contributors from an
+OWNERS file.
```

Comment:
I wonder if it should be 1 approver + 1 reviewer or approver.  That would match the workflow enforced for all PRs by prow right now.

Reply:
I could support that, sure.

Reply:
Updated